spec/models/evidence_summary_spec.rb

```diff
@@ -0,0 +1,123 @@
+RSpec.describe EvidenceSummary do
+  subject(:evidence_summary) { described_class.new(document) }
+  let(:document) do
+    Mas::Cms::Document.new(1, { blocks: blocks }.merge(attributes))
+  end
+  let(:attributes) { {} }
+  let(:blocks) { [] }
+
+  describe '#title' do
+    let(:attributes) { { 'label' => 'Financial well being' } }
+
+    it 'returns label from document' do
+      expect(evidence_summary.title).to eq('Financial well being')
+    end
+  end
+
+  describe '#evidence_type' do
+    context 'when layout identifier is present' do
+      let(:attributes) { { 'layout_identifier' => 'insight' } }
+
+      it 'returns layout identifier' do
+        expect(evidence_summary.evidence_type).to eq('Insight')
+      end
+    end
+
+    context 'when layout identifier is not present' do
+      let(:attributes) { {} }
```

Comment:
is this needed since its the same as line 6?

Reply:
I added here as well because represent the context. Perhaps I need to remove the line 6 then since I am forcing in each context. Wdyt?